extensions/Int+Extension.swift

```diff
@@ -19,7 +19,8 @@ public extension Int {
     }
 
     // MARK: - methods
-    private func formattedString(style: NumberFormatter.Style, localeIdentifier: String) -> String {
+    private func formattedString(style: NumberFormatter.Style, localeIdentifier: String) -> String
+    {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/tdrk18/swift-extensions/blob/22739ce465911863e00c5827fb3de497ac87b246/extensions/Int+Extension.swift#L23"></span>Opening braces should be preceded by a single space and on the same line as the declaration.</td>
    </tr>
  </tbody>
</table>